### PR TITLE
Development

### PR DESCRIPTION
**Pull request by FlyingBird95**
Let's make a PR, so https://fmd-master.herokuapp.com works again

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Adding the extension to your Flask app is simple:
     dashboard.bind(app)
     
 ## Live Demo
-To view a live deployment of the Flask-MonitoringDashboard, check [this site](https://flask-monitoringdashboard.herokuapp.com/).
+To view a live deployment of the Flask-MonitoringDashboard, check [this site](https://flask-master.herokuapp.com/).
```
Comment (mircealungu) [inline README.md]: @FlyingBird95 is this correct? do we have flask master?

Comment (mircealungu) [reply README.md]: ah, i see below: it's fmd-master!

File: docs/developing.rst [hunk 2]
```diff
@@ -131,6 +131,13 @@ The following tools are used for helping the development of the Dashboard:
     corresponding feature has been merged into the development branch. It is recommended to often
     merge development into this branch, to keep the feature branch up to date.
 
+- **Heroku deployment**: The following branches are automatically deployed to Heroku. This is useful for quickly
+  testing, without running any code locally.
+
+   - **Master**: The master branch is deployed at: `<https://fmd-master.herokuapp.com>`_.
+   - **Development**: The development is deployed at: `<https://fmd-development.herokuapp.com>`_.
+   - **Pull requests**: Pull requests are also automatically build with a unique URL.
```
Comment (mircealungu) [inline docs/developing.rst]: cool! where can one find the url? that would be worth mentioning!

File: README.md (hunk 1 above)
Comment (FlyingBird95) [reply README.md]: Good spot!! Changed it

File: docs/developing.rst (hunk 2 above)
Comment (FlyingBird95) [reply docs/developing.rst]: The url is automatically posted in the PR thread.

**Comment by FlyingBird95** [comment]
@mircealungu 

FYI for every PR that is created now, you'll see something like this:

Screenshot 2020-03-18 at 15 33 14


Clicking the link, and you can see a live deployment of the FMD of that PR.

**Comment by mircealungu** [comment]
Cool! I've merged your PR!

**Pull request merged**